build.gradle

```diff
@@ -198,15 +198,20 @@ task clean(type: Delete) {
     delete rootProject.buildDir
 }
 
+
 detekt {
     // The version number is duplicated, please refer to plugins block for more details
-    version = "1.0.0.RC7-3"
-    profile("main") {
-        input = "$projectDir/components"
-        config = "$projectDir/config/detekt.yml"
-        filters = ".*test.*,.*/resources/.*,.*/tmp/.*"
-        output = "$projectDir/build/reports/detekt"
-        baseline = "$projectDir/config/detekt-baseline.xml"
+    version = "1.0.0.RC9.2"
+    input = files("$projectDir/components")
+    config = files("$projectDir/config/detekt.yml")
```

Comment:
We have just one config, so this could be `file` instead?

Reply:
detekt somehow expects a collection 🤷‍♂️ 

```
> Cannot cast object '/Users/skaspari/Projects/Mozilla/android-components/config/detekt.yml' with class 'java.io.File' to class 'org.gradle.api.file.FileCollection'
```